portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,15 +35,62 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private List<String> names;
+
+  private List<String> messages;
+
   @Override
   public void init(){
     names = new ArrayList<>();
     names.add("Obi");
+
+    messages = new ArrayList<>();
+    messages.add("Hello! How are you");
+    //messages.add("You got this!");
+    //messages.add("Nice to meet you.");
   }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String name = names.get(0);
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello " + name + ". Welcome to JavaServlets");
+
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
can this move to init method or constructor?  Typically we try to minimize work done during actual request.